vcpkg.json

```diff
@@ -42,6 +42,7 @@
         "name": "libxmp",
         "default-features": false
     },
-    "cjson"
+    "cjson",
+    "fftw3"
```

Comment:
I measured sfx/rumble precache time and on my system, fftw3 is actually slower (10-run average):

```
no fft         1662 us
pffft          7279 us
fftw3         10546 us
fftw3 (avx2)   9200 us
```

There are benchmarks [here](https://github.com/project-gemmi/benchmarking-fft) with similar results. The "real-to-complex (1d-r.cpp)" test is the same as our use case.

So before I fight the build system any longer, should I roll back to pffft?

Reply:
Sure pffft is fine to me.

Reply:
Whatever means less hassle to you. But could you please move the library into the `third-party` directory?

Reply:
> Whatever means less hassle to you. But could you please move the library into the `third-party` directory?

Fixed here: https://github.com/fabiangreffrath/woof/pull/1876/commits/e92d72791efb909c0502206af2274f23ce171741